### DISCUSSION TITLE
How do I get many stars for my repository?

### DISCUSSION THREAD

**Discussion by defuj** [community/community#36095] [Repositories] [answered]
I want to get Starstruck Achievement on my Github account, I heard from my friend that he should get at least 16 stars on a project. https://github.com/defuj/def-helper

**Comment by radar-blip** [comment]
Your friend is right, you need 16 stars on a repository to get Starstruck Achievement Badge.

Here is some more info: [Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements)

**Reply by defuj** [reply]
Haha okay

**Comment by DSDmark** [comment, marked as answer]
Hey,
i m not so professional about that but 
I think it nice things to add **Social preview** it also affect your projects and first things is **topics**
for example, if you want to get stars write the topic which is use for many times in other word people search that specific topic to search particular thing and you can use unique topic which is only for your project like if someone search that topic only your repo should open

**Reply by defuj** [reply]
Oh, I see. Until now I didn't think about doing that.

**Reply by DSDmark** [reply]
And also I see your project it's great, but I think i m not sure but if you have time then you can try to add some **GIF** image in your **README** it's easy to understand other developer by **GIF** and try to not do long describe about your project in README try to make simple and also add some **Badges** on you project it's also look cool and contain small info about your projects
again, i m not so professional about that so, you can use these as a tips if you like

**Reply by ossanna16** [reply]
Great response @DSDmark! Thank you for helping! 🚀

**Reply by DSDmark** [reply]
Hey @ossanna16,
thanks for saying that, I'm glad to hear it.

**Reply by ossanna16** [reply]
Of course! Love seeing friendly and helpful users in the forum! Well done!
